org.btchd.btchd-qt.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
FYI ideally you should maintain it upstream.

Reply:
I asked but the owner is opposed to it for whatever reason.